dockers/docker-platform-monitor/docker_init.j2

```diff
@@ -105,9 +105,9 @@ confvar="{\"HAVE_SENSORS_CONF\":$HAVE_SENSORS_CONF, \"HAVE_FANCONTROL_CONF\":$HA
 
 if [ -e $PMON_DAEMON_CONTROL_FILE ];
 then
-    sonic-cfggen -j $PMON_DAEMON_CONTROL_FILE -a "$confvar" -t $SUPERVISOR_CONF_TEMPLATE > $SUPERVISOR_CONF_FILE
+    sonic-cfggen -d -j $PMON_DAEMON_CONTROL_FILE -a "$confvar" -t $SUPERVISOR_CONF_TEMPLATE > $SUPERVISOR_CONF_FILE
```

Comment:
why did you add the new option '-d' ?

Reply:
admin@sonic:~$ sudo sonic-cfggen -h
Render configuration file from minigraph data and jinja2 template.

  -d, --from-db         read config from configdb
  Basically need to pass the argments to jinja file which can help is decide whether to initiate this daemon, according to 'DualToR' type